src/test/java/com/sleekbyte/tailor/functional/yaml/YamlConfigurationTest.java

```diff
@@ -293,8 +293,8 @@ private void runTest(String[] command) throws UnsupportedEncodingException {
 
         String[] msgs = outContent.toString(Charset.defaultCharset().name()).split(NEWLINE_REGEX);
 
-        // Skip first three lines for file header, last two lines for summary
-        msgs = Arrays.copyOfRange(msgs, 3, msgs.length - 2);
+        // Skip first five lines for progress and file header, last two lines for summary
```

Comment:
Some places have 5 line headers and some have 4 line headers.  It might be good to give the example of what the header and summary look like in order to avoid any confusion.  What do you think?